build.cake

```diff
@@ -18,8 +18,6 @@ var ErrorDetail = new List<string>();
 var version = "3.6.0";
 var modifier = "";
 
-var isCompactFrameworkInstalled = FileExists(Environment.GetEnvironmentVariable("windir") + "\\Microsoft.NET\\Framework\\v3.5\\Microsoft.CompactFramework.CSharp.targets");
-
 //Find program files on 32-bit or 64-bit Windows
 var programFiles = Environment.GetEnvironmentVariable("ProgramFiles(x86)") ?? Environment.GetEnvironmentVariable("ProgramFiles");
 var isSilverlightSDKInstalled = FileExists(programFiles  + "\\MSBuild\\Microsoft\\Silverlight\\v5.0\\Microsoft.Silverlight.CSharp.targets");
```

Comment:
Suggestion: is it worth removing all the `isSilverlightInstalled()` and `isWindows()` checks? The entire script is a no-op if either of these are false, which doesn't seem so useful. 😄 We could perhaps have a `Checks` task however, which produces a nice error message if silverlight isn't installed...maybe that's a separate PR.